test/utils/utils.go

```diff
@@ -566,3 +582,48 @@ func VerifyCfgMapOrSecretEnvVarExists(c client.Client, expectResource ExpectApiR
 		return false
 	}
 }
+
+func ApplyPrometheusCRD() error {
```

Comment:
Refer to comment above, but I think `InstallPrometheusOperator` method, already provides us with the CRD?

Reply:
Right, this is for the prometheus object. But in that case, can we just define it in Go and apply? instead of having it as YAML.

I would avoid having definitions for tests in YAML as much as possible here, as things might change between versions which lead to confusing broken tests